pages/common/john.md

```diff
@@ -0,0 +1,36 @@
+# john
+
+> Password cracker.
+> More information: <https://www.openwall.com/john/>.
+
+- Crack password hashes:
+
+`john {{path/to/hashes.txt}}`
+
+- Show passwords cracked:
+
+`john --show {{path/to/hashes.txt}}`
+
+- Display user's cracked password by user identifier from multiple files:
+
+`john --show --users={{0}} {{path/to/hashes*}} {{path/to/other/hashes*}}`
```

Comment:
What does `0` mean here? Also, I think `*` is redundant after `hashes*`.

Reply:
`0` is the user identifier.

For the path, `*` has its importance ; it emphasizes that the wildcard can be used to designate any file starting with "`hashes`"...

Reply:
I understand it's an identifier, but it's not explained in the example's description. Maybe `user_id` instead of `0` would be better?

If the hashes file can have any name, then I'd suggest to drop the wildcard anyway.

Reply:
By the sounds of things, a wildcard is a very commonly used option there, so perhaps keeping it would be a good idea.

I do agree though that `{{user_id}}` instead of `{{0}}` might aid clarity in this situation though.

Reply:
I've let the wildcard and I changed the `{{0}}` to `{{user_ids}}`. Note the `user_idS` as the option is well `--userS`...